src/lib/pipeline.js

```diff
@@ -20,33 +20,21 @@ const schema = require('../format/log-schema.json');
 const validator = new Ajv();
 const validate = validator.compile(schema);
 
-function log(log, severity = 'warn') {
+function errorLog(log, severity = 'warn') {
```

Comment:
IMHO this is misleading as it is actually capable of doing regular logging with `severity = 'log'`.

Reply:
My struggle is that the codebase is already full of "log", and I wanted to have a less generic name for this function. I do agree with you, I'm not 100% sure about this change.

Reply:
Hmm true makes sense..
Only thing I can think of right now is `consoleLog` ( <= but I can imagine how misleading this would sound to anyone haha) or `pipelineLog` but none are really great..
Maybe `loggingFunction` || `logFn` ?